src/Helpers/Process/Commands/Drush.php

```diff
@@ -20,6 +20,9 @@ public function getBaseCommand(): string {
   protected function getCommand(array $commands): array {
     $uri = $this->input->getParameterOption('--uri');
     $commands = parent::getCommand($commands);
+    if (!empty(shell_exec(sprintf("which %s", escapeshellarg('./vendor/bin/drush'))))) {
```

Comment:
@rajeshreeputra Let's avoid this. This first looks for command and then update it. We can do following.
We can update `getCommand()` method of `CommandBase.php` like following:
is_executable($this->getBaseCommand())
```suggestion
  if (!is_executable($this->getBaseCommand())) {
    // Use ExecutableFinder to look for command globally.
  }
```


and then in Drush.php we can do following :
```suggestion
  public function getBaseCommand(): string {
    return './vendor/bin/drush';
  }
```

Reply:
updated